README.md

```diff
@@ -1,2229 +0,0 @@
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
If you remove this unwanted README, add a proper that explains the project installation, build process and deployment solutions.